src/index.jsx

```diff
@@ -32,21 +31,20 @@ import Groups from './components/groups'
 
 const store = configureStore()
 
-//request main info for user
-if (store.getState().authenticated){
+// request main info for user
+if (store.getState().authenticated) {
   store.dispatch(whoAmI())
 } else {
-  // just commented for develop sign up form
   store.dispatch(unauthenticated())
 }
 
-import {bindRouteActions} from './redux/route-actions'
+import { bindRouteActions } from './redux/route-actions'
 
 const boundRouteActions = bindRouteActions(store.dispatch)
 
 ReactDOM.render(
   <Provider store={store}>
-    <ReduxRouter>
+    <Router history={browserHistory}>
```

Comment:
Does this change anything for the user? (It's not a tricky question, I'm just being curious.)